test/Jamfile.v2

```diff
@@ -133,7 +133,7 @@ test-suite "serialization" :
 
 if ! $(BOOST_ARCHIVE_LIST) {
     test-suite "serialization2" : 
```

Comment:
Below: I found that test_inclusion needs singleton_module::get_lock which is in the library, so I removed the no-lib link designation from the test and it resolved that issue.